frontend/src/metabase/core/components/Slider/Slider.unit.spec.tsx

```diff
@@ -23,21 +23,42 @@ describe("Slider", () => {
     expect(minInput).toHaveAttribute("max", "412");
   });
 
-  it("should call onChange with the new value on input change", () => {
+  it("should call onChange with the new value on mouseUp", () => {
     const spy = jest.fn();
     render(<Slider value={[10, 20]} onChange={spy} min={0} max={100} />);
 
     const minInput = screen.getByLabelText("min");
     const maxInput = screen.getByLabelText("max");
 
-    // would be nice to use userEvent when we upgrade to v14 so we mock drage events
+    // would be nice to use userEvent when we upgrade to v14 so we can mock drag events
     fireEvent.change(minInput, { target: { value: "5" } });
+    fireEvent.mouseUp(minInput);
     fireEvent.change(maxInput, { target: { value: "15" } });
+    fireEvent.mouseUp(maxInput);
 
     const [firstCall, secondCall] = spy.mock.calls;
 
     expect(firstCall[0]).toEqual([5, 20]);
-    expect(secondCall[0]).toEqual([10, 15]);
+    expect(secondCall[0]).toEqual([5, 15]);
+  });
+
+  // handling input via arrow keys
+  it("should call onChange with the new value on keyup", () => {
```

Comment:
Oh cool. Just saw this component also supports keyboard actions which are great for accessibility but having some visual cues to indicate which slider (thumb) is active would be even better.